### DISCUSSION TITLE
pass request body to update an instance vs create new instance

### DISCUSSION THREAD

**Discussion by samaanjafari** [sqlalchemy/sqlalchemy#11409] [Ideas]
### Describe the use case

i was getting to used to connect my database through orms instead of raw sql then i realize when i was writing my crud pathes the functionality of update post and creating are actually the same put pathing into sql queries differs for example: im using fastapi as my frame work  and psotgresql as data base and pydantic model for my validation and sqlalchemy when u wanna create an instance in your table you should pass it like this : 
models.Post(**post.model_dump())
which unpack the dict and pass every individual value to keys automatically but as i think it supposed to be same in update but i find out we do not have unpacket it anymore than we have pass the dictionary itself and it was really confusing why?! then i search and found out its just designed like this and do not have any reason for this , So why don't we can do excatly the same thing in update query?

### Databases / Backends / Drivers targeted

postgresql and psycopg2

### Example Use
```
CREATING:
    new_post = models.Post(**post.model_dump())
    db.add(new_post)
    db.commit()
    db.refresh(new_post)
    
UPDATING:
    post_query = db.query(models.Post).filter(models.Post.id == id)
    post = post_query.first()
    post_query.update(updated_post.model_dump())
    
    db.commit()
```

**Comment by CaselIT** [comment]
Hi,

You can use merge to achieve something similar without needing to first query the db for the id: https://docs.sqlalchemy.org/en/20/orm/session_api.html#sqlalchemy.orm.Session.merge
